capture-sdk/sdk/src/main/res/values-night/styles.xml

```diff
@@ -8,6 +8,7 @@
         <item name="colorSurface">@color/Dark_02</item>
         <item name="backgroundColor">@color/Dark_01</item>
         <item name="colorOnBackground">@color/Light_01</item>
+        <item name="colorSecondary">@color/Dark_03</item>
```

Comment:
[Suggestion] How I understand it from the Material design documentation the [material theme secondary color](https://material.io/design/color/the-color-system.html) is not the right fit for the separator color. From the documentation:
> Secondary colors are best for:
> 
> Floating action buttons
> Selection controls, like sliders and switches
> Highlighting selected text
> Progress bars
> Links and headlines

What do you think of setting the colors directly on the separators without using a theme color? 

I am afraid that by using `secondaryColor` we also affect text highlight color in EditTexts and other widgets in unintended ways.